app/models/volunteer.rb

```diff
@@ -0,0 +1,2 @@
+class Volunteer < ApplicationRecord
+end
```

Comment:
@gabezurita don't we have to create Volunteer model as well? I don't see separate ticket for creating table

Reply:
@rjisoo Nice catch!

@gabezurita Can you run `rake db:migrate` and push the change to `db/schema.rb`?

Reply:
Done!

Reply:
There were also changes to the fields we wanted for the volunteer model, so redid the scaffold to include those changes.